src/main/java/io/confluent/kafka/connect/datagen/DatagenConnectorConfig.java

```diff
@@ -45,6 +48,16 @@
 
   public DatagenConnectorConfig(ConfigDef config, Map<String, String> parsedConfig) {
     super(config, parsedConfig);
+    if (getSchemaString().length() > 0) {
+      if (getSchemaFilename().length() > 0 || getQuickstart().length() > 0) {
```

Comment:
It seems like this should really be a check that exactly one of quickstart, schema.file, or schema.string is set. I kept the check more loose to avoid possibly breaking existing demos. @ybyzek if you're ok with it I can switch to the more restrictive check.

Reply:
Agree.  More restrictive will be less ambiguous.

Reply:
done